xbmc/addons/AddonBuilder.h

```diff
@@ -45,7 +45,7 @@ class CAddonBuilder
   void SetChangelog(std::string changelog) { m_addonInfo.m_changelog = std::move(changelog); }
   void SetBroken(std::string broken) { m_addonInfo.m_broken = std::move(broken); }
   void SetPath(std::string path) { m_addonInfo.m_path = std::move(path); }
-  void SetLibName(std::string libname) { m_addonInfo.m_libname = std::move(libname); }
+  void SetLibName(std::string libname) { m_addonInfo.m_types[0].m_libname = std::move(libname); }
```

Comment:
This is a temporary for old cpluff where always only one lib name is. On new way can be this no more direct accessed.

For a binary can it be only one, but on Python is it mostly on different scripts.